### PR TITLE
ZEPPELIN-342 Ignite interpreter updated

### PR DESCRIPTION
**Pull request by agura**

### CODE REVIEW

File: ignite/src/main/resources/default-ignite-jdbc.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (felixcheung) [inline ignite/src/main/resources/default-ignite-jdbc.xml]: does this file need to be built/deployed with the jar? or in the interpreter/ignite directory?

Comment (agura) [inline ignite/src/main/resources/default-ignite-jdbc.xml]: In current implementation this file should be deployed with jar. I

File: ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java [hunk 2]
```diff
@@ -58,14 +58,14 @@ public void setUp() {
 
     IgniteConfiguration cfg = new IgniteConfiguration();
     cfg.setDiscoverySpi(discoSpi);
+    cfg.setPeerClassLoadingEnabled(true);
```
Comment (felixcheung) [inline ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java]: this is set in default-ignite-jdbc.xml right? would test be able to pick up this setting from there?

**Comment by felixcheung** [comment]
looks good

File: ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java (hunk 2 above)
Comment (agura) [inline ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java]: default-ignite-jdbc.xml needs for JDBC connection that based on Ignite client node. But from test code we start Ignite server node with different configuration.

**Comment by Leemoonsoo** [comment]
Loooks good to me. :+1:

**Pull request closed without merging**